taker-frontend/src/App.tsx

```diff
@@ -13,6 +13,7 @@ import * as React from "react";
 import { useState } from "react";
```

Comment:
The commit description doesn't say why you're using an alert box instead of a toast - it would be much nicer if it actually stated the reasoning you used in the PR comment - ie. that it's a persistent problem and toasts are used for transient notifications.